L1Trigger/L1TTrackMatch/interface/L1TkEtMissEmuAlgo.h

```diff
@@ -0,0 +1,152 @@
+#ifndef L1Trigger_L1TTrackMatch_L1TkEtMissEmuAlgo_HH
+#define L1Trigger_L1TTrackMatch_L1TkEtMissEmuAlgo_HH
+
+#include "DataFormats/L1TrackTrigger/interface/TTTrack_TrackWord.h"
+#include "DataFormats/L1TrackTrigger/interface/TTTypes.h"
+#include <numeric>
+
+#include <cmath>
+#include <iostream>
+#include <cstdint>
+#include <filesystem>
+#include <fstream>
+#include <ap_int.h>
+
+using namespace std;
+// Namespace that defines constants and types used by the EtMiss Emulation
+// Includes functions for writing LUTs and converting to integer representations
+namespace L1TkEtMissEmuAlgo {
+
+  const unsigned int N_chi2XYbits_{4};  //Taken from TTTrack_Word class
```

Comment:
Why not take all of these constants and typedefs from the TTTrack_TrackWord class?

Reply:
Fixed in latest commit